TTS-Common-Control-Policy.md

```diff
@@ -16,20 +16,20 @@ All GSA employees and contracts are subject to these policies.
 
 ## Roles and responsibilities
 
-The GSA Office of the Chief Information Security Officer, otherwise known as GSA Information Security, is responsible for ensuring that all employees and contracts are familiar and trained in these policies, and to ensure that appropriate procedures are established and tested. The Cloud Operations team is responsible for all procedures and implementations.
+The GSA Office of the Chief Information Security Officer, otherwise known as GSA Information Security, is responsible for ensuring that all employees and contracts are familiar and trained in these policies, and to ensure that appropriate procedures are established and tested. The TTS Technology Portfolio Director acts as the liaison between GSA Information Security and TTS products, including cloud.gov. The Cloud Operations team is responsible for all procedures and implementations within cloud.gov.
```

Comment:
```suggestion
The GSA Office of the Chief Information Security Officer, otherwise known as GSA Information Security, is responsible for ensuring that all employees and contracts are familiar and trained in these policies, and to ensure that appropriate procedures are established and tested. The TTS Technology Portfolio Director acts as the liaison between GSA Information Security and TTS programs, including cloud.gov. The Cloud Operations team is responsible for all procedures and implementations within cloud.gov.
```